tests/src/Functional/ModuleSettingsFormTest.php

```diff
@@ -124,6 +125,7 @@ public function testGroupManagementAccess() {
 
     // Test member access.
```

Comment:
Do group members need access to the Group Domain settings page?

Reply:
Ah the difference between the two users here is that one is a member with no role but permission to 'use group_sites_admin_mode' permission only, and the other has a 'microsites-admin' role.